gcp/appengine/frontend_handlers.py

```diff
@@ -240,7 +240,7 @@ def vulnerability_redirector(potential_vuln_id):
 def bug_to_response(bug, detailed=True):
   """Convert a Bug entity to a response object."""
   response = osv.vulnerability_to_dict(
-      bug.to_vulnerability(include_alias=detailed))
+      bug.to_vulnerability(include_source=detailed, include_alias=detailed))
```

Comment:
No need to show this actually, as it replicates the existing "Source" field on the vulnerabilities field. Can you please revert?

Reply:
Ah, Thanks for the heads up! I didn't noticed the duplication. I dropped the commit.